### PR TITLE
Fixed: navigation bar overlapping + Event button

### LINKED ISSUE #2337: navigation bar overlapping + Event button
+ Event button is overlapped by the navigation bar, this position of button should be responsive

Screenshot_1705662515

can i work on this issue

### PR DESCRIPTION
**Pull request by RunTerror**
***Floating action button position fixed***
Fixes #2337 

Screenshot_1705774522
Screenshot_1705774534

@palisadoes please review my pr

### CODE REVIEW

**Comment by Cioppolo14** [comment]
@literalEval @noman2002 Can you review this PR?

**Comment by RunTerror** [comment]
@literalEval @noman2002 please review my pr

File: lib/views/demo_screens/explore_events_demo.dart [hunk 1]
```diff
@@ -71,6 +71,10 @@ class DemoExploreEvents extends StatelessWidget {
       // else render refresh icon along with the list of searched events for exploration.
       body: Stack(
         children: [
+          SizedBox(
+            width: SizeConfig.screenWidth,
+            height: SizeConfig.screenHeight,
+          ),
```
Comment (noman2002) [inline lib/views/demo_screens/explore_events_demo.dart:77]: Remove this and use "Expanded"

**Review by noman2002 — CHANGES_REQUESTED** [review]

File: lib/views/demo_screens/explore_events_demo.dart (hunk 1 above)
Comment (RunTerror) [reply lib/views/demo_screens/explore_events_demo.dart:77]: can i use flex as Expanded is giving errors

**Review by noman2002 — CHANGES_REQUESTED** [review]
Fix the conflicting files.

**Comment by noman2002** [comment]
Closing due to inactivity.

**Pull request closed without merging**